pkg/apis/starlingx/v1/system_types.go

```diff
@@ -57,7 +57,7 @@ type CertificateInfo struct {
 // point to a Secret named by the system.
 func (in *CertificateInfo) DeepEqual(other *CertificateInfo) bool {
 	if other != nil {
-		return in.Type == other.Type
+		return (in.Type == other.Type) && (in.Secret == other.Secret)
```

Comment:
Per discussion, it appears the Secret value from the model does not align with the Secret value from sysinv, so this field would not be able to be used for comparing items